gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/GateConfig.groovy

```diff
@@ -83,7 +81,7 @@ import static retrofit.Endpoints.newFixedEndpoint
 @CompileStatic
 @Configuration
 @Slf4j
-@EnableConfigurationProperties(FiatClientConfigurationProperties)
+@Import(FiatAuthenticationConfig)
```

Comment:
I don't think you want this. IIRC, I introduced the `EnableFiatAutoConfig` annotation to pull `FiatAuthenticationConfig` in _except_ for Gate.

Reply:
I'll double check the enabled/disabled case but the fiat autoconfiguration was working ok in gate once I cleaned up the bootstrapping in FiatAuthenticationConfig and removed the duplicated beans from in here

Reply:
FWIW I appear to be getting ...

```
Parameter 2 of method fiatService in com.netflix.spinnaker.fiat.shared.FiatAuthenticationConfig required a bean of type 'retrofit.client.OkClient' that could not be found.

Action:
Consider defining a bean of type 'retrofit.client.OkClient' in your configuration.
```

when running this PR locally?

Went back to master and fixed up the `fiatPermissionEvaluator()` and was running again.

Reply:
I suspect you might have been running this before I force pushed https://github.com/spinnaker/gate/pull/553/commits/f53d2fa54ac9b7bfaca544b87eb01a12e4007949 which included bumping spinnaker-dependencies with the fiat version that supports this change

Reply:
👍